### PR TITLE
[Update] Docs: enhance readme, add Community and Contributions section #14

### LINKED ISSUE #14: Docs: enhance readme, add Community and Contributions section
Recently, we have added our Contributing.md file that guides a user on how to contribute. Add a section in the README.md named "Community and Contributions" that references the user to the contributing guidelines. Also, list all the contributors to the project in this section.

Example - 
Screenshot 2023-10-13 at 8 37 05 PM

### PR DESCRIPTION
**Pull request by Freakyab**
**Description**
- ... Added "Community and Contributions section" in readme file.
- ...Added contributors to the project in this section.

**Related issue(s)**
Resolves #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,19 @@ project.
 - Lazy loaded output display.
 - No error logs to the console.
 - Used Vercel for hosting.
+
+# [Community](https://github.com/Freakyab/sql-editor/blob/main/CONTRIBUTING.md?plain=1) and [Contributions](https://github.com/Freakyab/sql-editor/blob/main/CONTRIBUTING.md?plain=1)
```
Comment (janvi01) [inline README.md:61]: no need to add links in the heading, kindly remove
```suggestion
## Community and Contributions
```

File: README.md [hunk 2]
```diff
@@ -57,3 +57,19 @@ project.
...
+
+Welcome to SQL Online Editor! We are thrilled to have you here and appreciate your interest in contributing. This document provides guidelines on how to get started with this project, contribute code, and improve the project together.
+
+### Ways to contribute to the project:
+- Reporting bugs: If you find a bug in our project, please report it by opening a new issue on GitHub.
+- Suggesting new features: If you have an idea for a new feature, please submit a new issue on GitHub.
+- Fixing bugs: If you want to work on any issue/bug, comment on the issue and make a pull request.
+- Improving documentation: If you find any errors or omissions in our documentation, please create an issue and submit a pull request with your changes.
+
```
Comment (janvi01) [inline README.md:70]: You have pasted the content of CONTRIBUTING.md here, Please look at the screenshot example given in the issue. We don't want to add contribution guidelines in the readme, just want to reference the contributing.md file.

File: README.md [hunk 3]
```diff
@@ -57,3 +57,19 @@ project.
...
+### Contributors
```
Comment (janvi01) [inline README.md:71]:
```suggestion
### Thanks to these wonderful Contributors 🌻
```

**Review by janvi01 — CHANGES_REQUESTED** [review]

**Comment by janvi01** [comment]
@Freakyab Thanks for the PR. Kindly make the required changes.

> Note: while creating a PR, always create a new branch. Keep this in mind for the next time. Follow the best [open-source practices](https://contributing.md/how-to-contribute-to-open-source/).

**Comment by Freakyab** [comment]
please review the changes

**Review by janvi01 — APPROVED** [review]
LGTM 🚀

**Pull request merged**